nix/default.nix

```diff
@@ -6,23 +6,22 @@
 let
   source = sources.tezos;
   release-binaries = import ./build/release-binaries.nix protocols;
-  binaries = builtins.listToAttrs (map (meta: {
+in {
+  octez-binaries = builtins.listToAttrs (map (meta: {
     inherit (meta) name;
     value = pkgs.octezPackages.${meta.name} // { inherit meta; };
   }) release-binaries);
 
-  # Bundle the contents of a package set together, leaving the original attrs intact
-  bundle = name: pkgSet:
-    pkgSet // (pkgs.buildEnv {
-      inherit name;
-      paths = builtins.attrValues pkgSet;
-    });
-
-  artifacts = { inherit binaries; };
-  bundled = (builtins.mapAttrs bundle artifacts);
-
-in (d: with builtins;
-     listToAttrs (map (drv:
-       { name = drv; value = d.${drv}; }
-     ) (filter (name: substring 0 5 name == "octez") (attrNames d)))
-   ) bundled.binaries // { inherit (bundled) binaries; }
+  tezos-binaries = builtins.listToAttrs (map (meta:
```

Comment:
What do you think about `trace`ing a deprecation warning here that binaries were renamed and one should switch to `octez-*`?

Reply:
Since it used primarily in our CI, i don't see why we should do this

Reply:
We shouldn't, since warning already raises for aliases, but packages still should be called as tezos-* (to be consistent with ubuntu/fedora packaging)